src/index.html

```diff
@@ -1,13 +1,426 @@
+                satisfied
+                customers premium segment. We focus only on new buildings and high-level doskanalno know this area
+                better
+                than other
+                specialists.Working directly with developers - so have more useful information about the profitable
+                deals.
+              </p>
+
+              <p class="about__paragraph about__paragraph--important">
+                We will never be late!
+                <br>
+                Guaranteed secure transaction - 100%
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section gallery" id="gallery">
+        <div class="container">
+          <h2 class="page__section-title">
+            Gallery
+          </h2>
+
+          <div class="gallery__place place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-8">
+                  <div class="place__photo-container">
+                    <img
+                      src="./images/photos/1.jpg"
+                      alt="Place 1"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+
+                <div class="grid__item grid__item--desktop-9-12">
+                  <div class="place__photo-container">
+                    <img
+                      src="./images/photos/2.jpg"
+                      alt="Place 1"
+                      class="place__photo"
+                    >
+                  </div>
+
+                </div>
+              </div>
+            </div>
+            <div class="place__description">
+              <a href="https://goo.gl/maps/pHHoJ9dw8X8rPS8o9" class="place__link">Marina Palms</a>
+
+              / North Miami Beach, FL 33162
+            </div>
+          </div>
+
+          <div class="gallery__place place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-6">
+                  <div class="place__photo-container">
+                    <img
+                      src="./images/photos/3.jpg"
+                      alt="Place 1"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+
+                <div class="grid__item grid__item--desktop-7-12">
+                  <div class="place__photo-container">
+                    <img
+                      src="./images/photos/4.jpg"
+                      alt="Place 1"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="place__description">
+              <a href="https://goo.gl/maps/q1HnL3j4SYVipAAg7" class="place__link">Edition Residens</a>
```

Comment:
Youur location-related addresses / links / images do not open google maps in a new tab target="_blank". You have to change it everywhere

Reply:
done